code/campaignclassic/src/main/java/com/adobe/marketing/mobile/CampaignPushTrackerActivity.java

```diff
@@ -204,7 +204,7 @@ private void openApplication() {
     private void openUri(final String uri) {
         try {
             final Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(uri));
-            intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_SINGLE_TOP);
+            intent.setFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP | Intent.FLAG_ACTIVITY_CLEAR_TOP);
```

Comment:
`FLAG_ACTIVITY_NEW_TASK` is implicitly added since CampaignPushTrackerActivity's launch mode is `singleInstance`. Adding `FLAG_ACTIVITY_CLEAR_TOP` to ensure that all of the other activities (which might depend on data from launched activity) on top of launched activity will be closed.